test/bdd/fixtures/kms-rest/docker-compose.yml

```diff
@@ -12,8 +12,8 @@ services:
     image: ${KMS_REST_IMAGE}:latest
     environment:
       - KMS_REST_HOST_URL=0.0.0.0:8070
-      - KMS_REST_TLS_CACERTS=/etc/tls/ec-cacert.pem
```

Comment:
Were these two env vars here by mistake before? Doesn't look like those args existed

Reply:
They are leftovers from my previous refactoring and initially were blindly added during bootstrapping the project.